testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/cacheddc/CachedDcDomainTestCase.java

```diff
@@ -166,6 +167,7 @@ public void dcConfigChangePropagated() throws Exception {
         domainManager = DomainTestSupport.create(domainConfig);
         domainManager.start();
 
+        domainManager.getDomainSlaveLifecycleUtil().awaitHostController(System.currentTimeMillis());
```

Comment:
domainManager.start() should be doing this, via its call to DomainLifecycleUtil.start(). Was this not working somehow?